packages/fixed-decimal/src/FixedDecimal.ts

```diff
@@ -168,7 +170,14 @@ export class FixedDecimal {
    * @param decimalPlaces The number of decimal places the value has.
    */
   constructor(
-    value: FixedDecimalCtorArgs | bigint | number | string | BN,
+    value:
+      | FixedDecimalCtorArgs<T>
+      | T
```

Comment:
Suggestion: Is it too much to ask to use more descriptive names for the generic parameters? Reading through this, I 'm not sure what `T` or `K` represent without referencing back up to the top each time I see them.